app/models/user.rb

```diff
@@ -20,4 +20,7 @@ class User < ApplicationRecord
   scope :by_source_site, ->(source_site) { where(source_site: source_site) }
 
   enum gender: { male: 0, female: 1 }
+  enum notification_frequency: {
+    disabled: 0, immediate: 1, hourly: 2, daily: 3, weekly: 4
+  }, _suffix: :notifications
```

Comment:
This is just to keep these enum values in context because they could be shared between multiple enums. By default, the bang and predicate methods are in this format:

```ruby
user.immediate!
```

By setting the suffix, the API for dealing with notifications frequency is:

```ruby
user.immediate_notifications!
```

What do you think?